experiment/generate_tests.py

```diff
@@ -22,18 +22,56 @@
 
   $ ./experiment/generate_tests.py \
       --yaml-config-path=experiment/test_config.yaml \
-      --json-config-path=jobs/config.json
+      --json-config-path=jobs/config.json \
+      --prow-config-path=prow/config.yaml
 """
 
 import argparse
 import hashlib
 import json
 import os
-import yaml
+import ruamel.yaml as yaml
```

Comment:
PyYAML does not handle comments -- comments will be lost on a load-and-dump. And looks like PyYAML's development seems to have stalled.